syntax = "proto3";

// 定义一个服务
service HelloService {
  // 定义一个 RPC 方法
  rpc SayHello (HelloRequest) returns (HelloReply);
}

// 请求结构
message HelloRequest {
  string name = 1;
}

// 返回结构
message HelloReply {
  string message = 1;
}
